examples/angular_bazel_architect/README.md

```diff
@@ -0,0 +1,5 @@
+# Frontend
```

Comment:
Add a brief description of the example here and mention that the patch is still needed at this point?